### PR TITLE
Partial fix to adding notes to some notable strats

### PR DESCRIPTION
**Pull request by kjbranch**
A quick first pass at adding and improving notes on notable strats. 
- Added a note to most where they were missing,
- Fixed some to make them more clear or more correct. 

These quick notes can help the player have an idea what the strat means from a higher level. 
This should all be double checked and improved eventually.

### CODE REVIEW

File: region/maridia/inner-yellow.json [hunk 1]
```diff
@@ -237,20 +237,20 @@
                 {
                   "name": "Bug Room Spark Out Left with Hijump",
                   "notable": false,
-                  "note": "Charge below then climb up and carry the spark out of the left door.",
                   "requires": [
                     "canShinechargeMovementComplex",
                     "HiJump"
-                  ]
+                  ],
+                  "note": "Charge below then climb up and carry the spark out of the left door."
```
Comment (osse101) [inline region/maridia/inner-yellow.json:244]: Charge a spark right then run and jump up

File: region/maridia/inner-yellow.json [hunk 2]
```diff
@@ -237,20 +237,20 @@
...
                 {
                   "name": "Bug Room Spark Out Left HiJumpless",
                   "notable": true,
-                  "note": "Charge below then climb up and carry the spark out of the left door.",
                   "requires": [
                     "canShinechargeMovementComplex",
                     "canTrickyDashJump"
-                  ]
+                  ],
+                  "note": "Charge below then climb up and carry the spark out of the left door."
```
Comment (osse101) [inline region/maridia/inner-yellow.json:253]: Charge a spark right then run and jump up

File: region/maridia/inner-yellow.json [hunk 3]
```diff
@@ -263,12 +263,12 @@
...
                   "name": "Bug Room Charge Out Left with Hijump",
                   "notable": false,
-                  "note": "Charge below then climb up and carry the spark out of the left door.",
                   "requires": [
                     "canShinechargeMovementComplex",
                     "HiJump",
                     "canWalljump"
-                  ]
+                  ],
+                  "note": "Charge below then climb up and carry the spark out of the left door."
```
Comment (osse101) [inline region/maridia/inner-yellow.json:271]: Charge a spark left and use the last bit of platform to jump up to the ledge above

File: region/maridia/inner-yellow.json [hunk 4]
```diff
@@ -1370,7 +1370,7 @@
               ],
               "bypassStrats": [
                 {
-                  "name": "Botwoon Skip",
+                  "name": "Butterfly Room Botwoon Skip with Wall Ice Clip",
```
Comment (osse101) [inline region/maridia/inner-yellow.json:1373]: I don't know that calling it Botwoon skip makes sense anymore.  You are skipping a Draygon door lock.

File: region/norfair/east.json [hunk 5]
```diff
@@ -52,7 +52,8 @@
                       "canWalljump",
                       "SpaceJump"
                     ]}
-                  ]
+                  ],
+                  "note": "Kill the enemies and use the bottom to charge a spark. Quickly climb then spark through the left door."
```
Comment (osse101) [inline region/norfair/east.json:56]: Clear the shot block first too

File: region/norfair/east.json [hunk 6]
```diff
@@ -4230,18 +4232,22 @@
                   ]
                 },
                 {
-                  "name": "Double Chamber Hijumpless Walljump",
+                  "name": "Double Chamber HiJumpless Wall Jump",
                   "notable": true,
                   "requires": [
                     "h_heatProof",
                     "canInsaneWalljump",
+                    "canIframeSpikeJump",
```
Comment (osse101) [inline region/norfair/east.json:4240]: I don't like this add but it is hard to dispute.

File: region/norfair/west.json [hunk 7]
```diff
@@ -2884,34 +2884,28 @@
...
-                      "requires": [
+                          "shinesparkFrames": 84,
+                          "excessShinesparkFrames": 37
+                        }},
+                        {"canComeInCharged":{
+                          "fromNode": 4,
+                          "inRoomPath": [4, 6],
+                          "framesRemaining": 180,
+                          "shinesparkFrames": 84,
+                          "excessShinesparkFrames": 37
```
Comment (osse101) [inline region/norfair/west.json:2895]: I count about 44

File: region/lowernorfair/east.json [hunk 8]
```diff
@@ -1028,7 +1028,8 @@
                       "id": "A",
                       "requires": ["h_canDestroyBombWalls"]
                     }
-                  ]
+                  ],
+                  "note": "Freeze the pirates to use as platforms to get up through the bomb blocks."
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/lowernorfair/east.json:1032]: It's looking like only standing pirate hitboxes are really tall

**Pull request merged**